libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -2430,8 +2430,10 @@ def embed_bootloader(self, f):
 
         bp = self.bootloader_path()
         if not os.path.exists(bp):
-            self.error("Bootloader (%s) does not exist and AP_BOOTLOADER_FLASHING_ENABLED" %
-                       (bp,))
+            self.error('''Bootloader (%s) does not exist and AP_BOOTLOADER_FLASHING_ENABLED
+Please run: Tools/scripts/build_bootloaders.py %s
+''' %
+                       (bp,os.path.basename(os.path.dirname(args.hwdef[0]))))
```

Comment:
wrong PR?

Reply:
i've moved it to a new PR: https://github.com/ArduPilot/ardupilot/pull/25835